### ISSUE TITLE
Allow links between comments on a single paper

### ISSUE THREAD

**Issue by freelanceastro** [openjournals/theoj#134]
Apparently something similar has already been implemented on the Journal of Brief Ideas; let's copy that if we can.

**Comment by marcrohloff** [comment]
It might be simpler to change our path style to use pushState paths. i.e.
   /papers/abcd#comment1
Rather than try and do
   /#/papers/abcd/comment1

See issues #127 and #76.

**Comment by marcrohloff** [comment]
I'm not sure that we should do this in this milestone. I think that it is reasonably complex

**Comment by freelanceastro** [comment]
I agree, unless @stuartlynn really thinks it's simple to export the solution from the Journal of Brief ideas.

**Comment by marcrohloff** [comment]
We might want to think about what a comment link would look like. Any ideas?

**Comment by marcrohloff** [comment]
I couldn't find a way to comment there at all. Did I miss something?

**Comment by freelanceastro** [comment]
No idea. @stuartlynn, what did you have in mind?

**Comment by stuartlynn** [comment]
There are no comments on the JBI but when you create a new idea you can type # and then it gives suggestions for other ideas and then on enter creates the markdown for the link to them. We could do something similar here.

<iframe src="http://gfycat.com/ifr/NippyCoarseBaldeagle" frameborder="0" scrolling="no" width="452" height="450" style="-webkit-backface-visibility: hidden;-webkit-transform: scale(1);" ></iframe>
 
 That link could scroll the window to that issue or open another single issue window with that issue in it.
